### PR TITLE
feat(rdb): attribute init_settings

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: docs/resources/rdb_instance.md [hunk 1]
```diff
@@ -9,9 +9,7 @@ description: |-
 Creates and manages Scaleway Database Instances.
 For more information, see [the documentation](https://developers.scaleway.com/en/products/rdb/api).
 
-## Examples
```
Comment (Codelax) [inline docs/resources/rdb_instance.md:12]: You could let `Examples` title before examples

File: docs/resources/rdb_instance.md [hunk 2]
```diff
@@ -129,6 +152,10 @@ The following arguments are supported:
 
 - `backup_same_region` - (Optional) Boolean to store logical backups in the same region as the database instance.
 
+- `init_settings` - (Optional) Map of engine settings to be set at database initialisation.
+
+~> **Important:** Updates to `init_settings` will recreate the Database Instance.
+
 - `settings` - (Optional) Map of engine settings to be set. Using this option will override default config. Available settings for your engine can be found on scaleway console or fetched using [rdb engine list route](https://developers.scaleway.com/en/products/rdb/api/#get-1eafb7)
```
[the remark this answers is not in the document]
Comment (Monitob) [reply docs/resources/rdb_instance.md:159]: In fact, There a different flags.

File: scaleway/resource_rdb_instance_test.go [hunk 3]
```diff
@@ -198,6 +198,41 @@ func TestAccScalewayRdbInstance_Settings(t *testing.T) {
...
+	defer tt.Cleanup()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy:      testAccCheckScalewayRdbInstanceDestroy(tt),
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource scaleway_rdb_instance main {
+						name = "test-rdb"
```
Comment (Codelax) [inline scaleway/resource_rdb_instance_test.go:212]: Can this create a conflict with other tests ?

Comment (Monitob) [reply scaleway/resource_rdb_instance_test.go:212]: Yes, I'm gonna rename it

**Pull request merged**